simapp/simd/cmd/root.go

```diff
@@ -98,8 +98,25 @@ func initAppConfig() (string, interface{}) {
 		WASM WASMConfig `mapstructure:"wasm"`
 	}
 
+	// Optionally allow the chain developer to overwrite the SDK's default
+	// server config.
+	srvCfg := serverconfig.DefaultConfig()
+	// The SDK's default minimum gas price is set to "" (empty value) inside
+	// app.toml. If left empty by validators, the node will halt on startup.
+	// However, the chain developer can set a default app.toml value for their
+	// validators here.
+	//
+	// In summary:
+	// - if you leave srvCfg.MinGasPrices = "", all validators MUST tweak their
+	//   own app.toml config,
+	// - if you set srvCfg.MinGasPrices non-empty, validators CAN tweak their
+	//   own app.toml to override, or use this default value.
+	//
+	// In simapp, we set the min gas prices to 0.
+	srvCfg.MinGasPrices = "0stake"
```

Comment:
Had a chat with @aaronc last evening about this, we were looking around `Viper.SetDefault`, but this morning I noticed this new `initAppConfig` function introduced by @anilCSE in #9550. And it seems to me like the perfect place for chain developers to set a default min gas price.

cc. @marbar3778

Reply:
this is awesome!! We should add a note in the docs that @anilCSE added that this is possible.